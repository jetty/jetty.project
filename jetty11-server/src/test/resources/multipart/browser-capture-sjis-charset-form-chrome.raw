------WebKitFormBoundaryDHtjXxgNUcgLjcKs
Content-Disposition: form-data; name="_charset_"

Shift_JIS
------WebKitFormBoundaryDHtjXxgNUcgLjcKs
Content-Disposition: form-data; name="japanese"

Œ’Ž¡
------WebKitFormBoundaryDHtjXxgNUcgLjcKs
Content-Disposition: form-data; name="hello"

ƒƒƒ†&#25094;ƒ^
------WebKitFormBoundaryDHtjXxgNUcgLjcKs--
