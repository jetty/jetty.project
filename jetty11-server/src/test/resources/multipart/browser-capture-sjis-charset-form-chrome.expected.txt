Request-Header|Accept|text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8
Request-Header|Accept-Encoding|gzip, deflate, br
Request-Header|Accept-Language|en-US,en;q=0.9
Request-Header|Cache-Control|max-age=0
Request-Header|Connection|keep-alive
Request-Header|Content-Length|354
Request-Header|Content-Type|multipart/form-data; boundary=----WebKitFormBoundaryDHtjXxgNUcgLjcKs
Request-Header|Cookie|visited=yes
Request-Header|DNT|1
Request-Header|Host|localhost:9090
Request-Header|Origin|http://localhost:9090
Request-Header|Referer|http://localhost:9090/sjis-form-charset.html
Request-Header|Upgrade-Insecure-Requests|1
Request-Header|User-Agent|Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.162 Safari/537.36
Parts-Count|3
Part-ContainsContents|_charset_|Shift_JIS
Part-ContainsContents|japanese|健治
Part-ContainsContents|hello|ャユ&#25094;タ